extension/public/static/manifest.json

```diff
@@ -16,7 +16,7 @@
     }
   ],
   "web_accessible_resources": ["lyraApi.min.js"],
-  "page_action": {
+  "browser_action": {
```

Comment:
this keeps the Lyra extension icon enabled on every page, unconditionally

Reply:
page_action was looking for a condition to load the popup, but I removed the condition yesterday. whoops!